cli/azd/cmd/deploy.go

```diff
@@ -227,7 +227,10 @@ func (da *deployAction) Run(ctx context.Context) (*actions.ActionResult, error)
 		if alphaFeatureId, isAlphaFeature := alpha.IsFeatureKey(string(svc.Host)); isAlphaFeature {
```

Comment:
Can you also please take this request into consideration? https://github.com/Azure/azure-dev/issues/690

Reply:
Will do that in a follow up PR and before we graduate this out of alpha.